python/foglamp/services/core/service_registry/monitor.py

```diff
@@ -31,60 +30,61 @@ class Monitor(object):
     _DEFAULT_PING_TIMEOUT = 1
     """Timeout for a response from any given micro-service"""
 
-    _logger = None  # type: logging.Logger
+    _DEFAULT_MAX_ATTEMPTS = 15
 
-    def __init__(self):
+    _logger = None
 
-        self._logger = logger.setup("SMNTR", level=logging.INFO)
+    def __init__(self):
+        self._logger = logger.setup(__name__, level=20)
```

Comment:
logging.INFO is far better than using 20

Reply:
logging.INFO is posing some issues if used with "from foglamp.common import logger". Do not know why. Can you pl check this at your end?